sentry-android-core/src/test/java/io/sentry/android/core/AppStartStateTest.kt

```diff
@@ -77,4 +77,16 @@ class AppStartStateTest {
 
         assertEquals(400, sut.appStartInterval)
     }
+
+    @Test
+    fun `getAppStartInterval returns null if more than 60s`() {
```

Comment:
`m`: We could add another test with the max app start duration of 59999 ms to validate it is working correctly.

Reply:
I don't think this is unnecessary, we already test above and lower the boundary.
The comparison is a simple `>=` with a long value rather than some more complicated calculation.
I can do it but sounds like not needed, your call.

Reply:
If someone changes the MAX_APP_START_MILLIS to 10000000, no test will fail. Up to you.